### DISCUSSION TITLE
Dependabot: Include all vulnerable repositories in weekly digest

### DISCUSSION THREAD

**Discussion by Dunedan** [community/community#48116] [Code Security]
### Select Topic Area

Product Feedback

### Body

Currently the weekly Dependabot digest emails only contain vulnerabilities for up to 10 repositories. That's not made transparent in the email itself and might result in a wrong picture of the number of  repositories affected by security vulnerabilities.

The only place this limit is mentioned seems to be the documentation: https://docs.github.com/en/code-security/dependabot/dependabot-alerts/configuring-notifications-for-dependabot-alerts#configuring-notifications-for-dependabot-alerts

It'd be great if these digest emails would include security vulnerabilities for all repositories or at least a note stating that not all repositories are included in the email.

**Comment by erinhav** [comment]
👋 Hi @Dunedan. PM for Dependabot Alerts here. That's _great_ feedback -- this setting is definitely misleading. We'll try to pick up a fix shortly!
